pkgs/by-name/ta/tabby/package.nix

```diff
@@ -1,36 +1,76 @@
 { lib
+, rustPlatform
 , fetchFromGitHub
+, stdenv
+
 , gcc12
-, cmake
 , git
 , openssl
 , pkg-config
 , protobuf
-, rustPlatform
-, addOpenGLRunpath
-, cudatoolkit
-, nvidia ? true
+
+, llama-cpp
+# one of `[ null "rocm" "cuda" ]`
+, acceleration ? null
```

Comment:
I would either
- not have that configurable and just use config.cudaSupport and rocmsupport
- or at least have the defaults being picked up by the config.

Reply:
See comment in other thread

https://github.com/NixOS/nixpkgs/pull/292873#discussion_r1510808797